src/Swarm/TUI/Inventory/Sorting.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+-- |
+-- Module      :  Swarm.TUI.Inventory.Sorting
+-- Copyright   :  Brent Yorgey
+-- Maintainer  :  byorgey@gmail.com
+--
+-- SPDX-License-Identifier: BSD-3-Clause
+module Swarm.TUI.Inventory.Sorting (
+  InventorySortOptions (..),
+  InventorySortDirection (..),
+  InventorySortOrder (..),
+  cycleSortOrder,
+  cycleSortDirection,
+  sortInventory,
+  renderSortMethod,
+) where
+
+import Control.Lens (view)
+import Data.List (sortBy)
+import Data.Ord (comparing)
+import Data.Text qualified as T
+
+import Algorithms.NaturalSort (sortKey)
+
+import Swarm.Game.Entity as E
```

Comment:
I think what we've agreed on most recently is to list all the imports in one big block with no blank lines in between, because these days imports can just be handled automatically by your favorite editor/IDE, so there's no particular reason any more to separate them out into logical groups to make them easier for humans to read.  I don't know if we've actually converted all the source files in the repo to use that format or whether there are still some that have separate blocks of imports.